tests/test_core.py

```diff
@@ -8808,6 +8806,12 @@ def test_minimal_runtime_emscripten_get_exported_function(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_get_exported_function')
 
 
+  @node_pthreads
+  def test_binaryen_2170_emscripten_atomic_cas_u8(self, js_engines):
+    self.emcc_args += ['-s', 'USE_PTHREADS=1']
```

Comment:
the decorator already adds USE_PTHREADS=1 so I think this line is unnecessary.

Reply:
Hmm, I think I'd rather have that be in there to be explicit. I don't like applying hidden options.

Reply:
It's not hidden :smile: the decorator says `node_pthreads` so it enables both node and pthreads.

Or did I misunderstand what you meant?

Reply:
Yeah I guess :) I meant it's nice to read explicitly what non-default args a test adds.

Reply:
I suppose, but I think I prefer to be concise personally. Anyhow, we should at least be consistent, and either change all other tests to how you prefer, or make this test like the existing ones.

@sbc100 do you have a preference?

Reply:
I admit I didn't know that `@node_pthreads` meant that.     I don't think its a big deal.

It would slightly cleaner to do `self.set_setting('USE_PTHREADS')` though.. this amounts to the same thing but gives the test framework a little more visibility of you intent.

Reply:
If you both prefer the visibility than that sgtm - please change the other test or two that use that annotation to be the same.